lib/Page.js

```diff
@@ -26,6 +26,12 @@ const {Keyboard, Mouse, Touchscreen} = require('./Input');
 const Tracing = require('./Tracing');
 const {helper, debugError} = require('./helper');
 
+function writeFileAsync(...options) {
```

Comment:
@aslushnikov , should this be added in helper? I could use it in https://github.com/GoogleChrome/puppeteer/pull/881.

Reply:
As it seems to me, in a certain file/folder "utils" or "helpers" it is necessary to make the function promisify (as [here](https://github.com/CheerlessCloud/puppeteer/blob/master/utils/doclint/SourceFactory.js#L150)) and use it to create wrappers of the fs methods or some other in local module scope (as [here](https://github.com/CheerlessCloud/puppeteer/blob/master/utils/doclint/SourceFactory.js#L20)). Or we can add some external dependency, such as Bluebird or [pify](https://github.com/sindresorhus/pify).
IMHO, I, as a person external to the project, can not make such decisions. And I do not want to rewrite the code, so let's work out solution and I will implement it in this PR. This is almost my first PR to open source project, so I may not know some rules, sorry =)